libs/sdk-ui-pivot/src/impl/resizing/agGridColumnSizing.ts

```diff
@@ -910,7 +910,7 @@ export const getAutoResizedColumns = (
         separators: any;
     },
 ): IResizedColumns => {
-    if (tableDescriptor && gridApi && columnApi && execution) {
+    if (tableDescriptor && gridApi && columnApi && execution && containerRef) {
```

Comment:
looks good. one small thing. please modify signature so that containerRef is Element | null